### PR TITLE
Longtext input has x on top of scrollbar

### PR DESCRIPTION
**Pull request by PoornimaVKrishnan**
This PR is related to 
https://github.com/informatics-isi-edu/chaise/issues/2318

It focuses on adding a padding when there is a scrollbar in the textarea which avoids overlapping between the text, cross button and scrollbar

### CODE REVIEW

File: src/utils/input-utils.ts [hunk 1]
```diff
@@ -327,3 +327,10 @@ export const VALIDATE_VALUE_BY_TYPE: {
   'timestamptz': timestamptzFieldValidation,
   'color': colorFieldValidation,
 };
+/**
+ * Function to check if there is scrollbar for textarea
+ */
+export const hasScrollbar = (element: any) => {
```
Comment (RFSH) [inline src/utils/input-utils.ts:333]: Change this to `hasVerticalScrollbar`

File: src/assets/scss/_input-switch.scss [hunk 2]
```diff
@@ -7,6 +7,14 @@
     height: auto;
   }
 
+  .has-scrollbar {
+    padding-right: 2rem !important;
```
Comment (RFSH) [inline src/assets/scss/_input-switch.scss:11]: Add a comment above this rule saying that you're adding this rule to increase the space to accommodate for scrollbar

File: src/assets/scss/_input-switch.scss [hunk 3]
```diff
@@ -7,6 +7,14 @@
...
+  }
+
+  .has-scrollbar-clear {
+    margin-right: 10px;
```
Comment (RFSH) [inline src/assets/scss/_input-switch.scss:15]: Add a similar comment for this as well

**Review by RFSH — CHANGES_REQUESTED** [review]
Your changes looks good overall. I asked for some small changes. By the way, In the issue I forgot to mention json-field.tsx. So make sure to apply a similar change to the `textarea` in there as well.

**Comment by PoornimaVKrishnan** [comment]
@RFSH 
Addressed all comments

**Review by RFSH — APPROVED** [review]

**Pull request merged**